mcu-interface/src/can/mod.rs

```diff
@@ -1,2 +1,6 @@
+use std::time::Duration;
+
 pub mod canfd;
 pub mod isotp;
+
+const RX_TIMEOUT: Duration = Duration::from_millis(1500);
```

Comment:
what do you think about renaming to `ACK_RX_TIMEOUT`?

Reply:
Agreed